process/errors.go

```diff
@@ -28,6 +28,9 @@ var ErrNilCryptoComponentsHolder = errors.New("nil crypto components holder")
 // ErrNilDataComponentsHolder signals that a nil data components holder was provided
 var ErrNilDataComponentsHolder = errors.New("nil data components holder")
 
+// ErrNilRunTypeComponentsHolder signals that a nil run type components holder was provided
+var ErrNilRunTypeComponentsHolder = errors.New("nil run type components holder")
```

Comment:
Let's have all these new errors from this file only in `errors/errors.go`

Reply:
removed one error and moved another to errors/errors.go